b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItem.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import CodeIcon from "@rsuite/icons/Code";
+import React, { useCallback, useEffect, useState } from "react";
+import { Nav, Panel } from "rsuite";
+import General from "./roleItemDetailsSection/general";
+import decodeGetRole from "../../../../../../util/apiDecode/settings/role/decodeGetRole";
+import AccordianItemHeader from "../../../../../common/accordianItemHeader";
+import JsonDisplay from "../../../../../common/jsonDisplay";
+
+/**
+ * 
+ * @param prop - `session`, `id`, `roleUri`
+ * 
+ * @returns role item componet
+ */
+export default function RoleItem(prop) {
+
+    const { session, id, roleUri } = prop;
+
+    const [ roleDetails, setRoleDetails ] = useState({});
+    const [ activeKeyNav, setActiveKeyNav ] = useState("1");
+
+    const fetchData = useCallback(async () => {
+        const res = await decodeGetRole(session, roleUri);
+
+        setRoleDetails(res);
+    }, [ session, roleUri ]);
+
+    useEffect(() => {
+        fetchData();
+    }, [ fetchData ]);
+
+    const activeKeyNavSelect = (eventKey) => {
+        setActiveKeyNav(eventKey);
+    };
+
+    const roleItemDetailsComponent = (activeKey) => {
+        switch (activeKey) {
+            case "1":
+
+                return <General session={ session } roleDetails={ roleDetails } fetchData={ fetchData } />;
+                // case "2":
+
+            //     return <Settings session={session} idpDetails={idpDetails} />;
```

Comment:
Why has this been commented out?

Reply:
please provide a reason.

Reply:
This is already fixed from the commit https://github.com/Achintha444/samples-is/commit/0db49b892ae09126a0bb153033074fa0a06ec2ba#diff-48d3a4aa1e8e5111ed172799200ba92ea05adc72937247d83c550ee7ce702e69